### PR TITLE
Test fixes

### PR DESCRIPTION
**Pull request by e10harvey**

### CODE REVIEW

File: opencsp/common/lib/render/figure_management.py [hunk 1]
```diff
@@ -104,8 +134,8 @@ def tile_figure(
     ul_y = size_y_pixels * y_idx
 
     # Create figure.
-    # fig = plt.figure(constrained_layout=True).subplots(5, 5)
-    fig = plt.figure(name, figsize=(size_x, plot_size_y))
+    # fig = figure(constrained_layout=True).subplots(5, 5)
```
Comment (braden6521) [inline opencsp/common/lib/render/figure_management.py:137]: If you think it's a good idea, could we remove this commend out line? For the sake of preserving what we know works and minimizing the likelihood of someone accidently bringing up this error again.

Comment (e10harvey) [reply opencsp/common/lib/render/figure_management.py:137]: That's OK with me. @bbean23 what do you think?

Comment (bbean23) [reply opencsp/common/lib/render/figure_management.py:137]: I think it'd be good to remove the commented out line. Note that the `figure()` method name changed again in #15, so you should probably pull again.

**Comment by e10harvey** [comment]
Closing in favor of #40.

**Pull request closed without merging**